tests/network/avg_speed/conftest.py

```diff
@@ -9,8 +9,8 @@
 
 @pytest.fixture(name="avg_speed_csv")
 def get_avg_speed_csv_filepath() -> Iterator[Path]:
-    _ini_file = test_data.joinpath("network", "test_read_avg_speed", "avg_speed.csv")
-    yield _ini_file
+    _csv_file = test_data.joinpath("network", "test_read_avg_speed", "avg_speed.csv")
+    yield _csv_file
```

Comment:
why not checking beforehand that the file exists?

Reply:
Done in #504